shell/components/DetailTop.vue

```diff
@@ -97,6 +99,10 @@ export default {
       return this.value?.filteredSystemLabels;
     },
 
+    filteredLabels() {
```

Comment:
The component is full of logic on its own on manipulating the `value`, do you want to refactor it now?

Reply:
The component generically handles input from `value` and converts it into a specific format that it needs to display the information. Moving type specific code into the model will maintain that generic pattern

Reply:
The `value` computation from the model is already inside the component, which is not abstracted, I am not so sure to follow.
To avoid labels of certain types is part of the `value` computation like the others. Why is this an exception?

Reply:
`DetailTop` is a generic component, it has no reference to type specific code (PSALabelsNamespaceVersion) only generic methods exposed by models/classes. 

The model for namespaces can be updated to supply the type specific information that is accessed by the generic component in a generic way. Depending on how the Labels component works this can be the `get labels` getter in namespaces. If you still need the original labels list for other components, it looks like you can add a `get filteredSystemLabels` to namespaces that will contain the PSA label filtering

Reply:
I'm trying to understand how to move the config into the model, but:
- We have already a config for the edit mode on top
- Labels is not a model, just a list from the value
- The value is not always namespace, so if I move the configuration it's not always present

What should I do in this case?